components/brave_wallet_ui/components/shared/create-network-icon/index.tsx

```diff
@@ -124,7 +124,7 @@ export const CreateNetworkIcon = ({ network, marginRight, size }: Props) => {
     >
       <NetworkIcon
         size={size}
-        icon={isRemoteURL ? `chrome://image?${networkImageURL}` : networkIcon}
+        icon={isRemoteURL ? `chrome://image?${encodeURIComponent(networkImageURL)}` : networkIcon}
```

Comment:
I'm not super familiar with `chrome://image` but the fact we're passing a queryString here seems to indicate that this should be URI encoded. Let's let the wallet team confirm though